src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -402,25 +399,6 @@ private static unsafe int AlpnServerSelectCallback(IntPtr ssl, out byte* outp, o
             return Ssl.SSL_TLSEXT_ERR_NOACK;
         }
 
-        private static void UpdateCAListFromRootStore(SafeSslContextHandle context)
-        {
-            using (SafeX509NameStackHandle nameStack = Crypto.NewX509NameStack())
-            {
-                //maintaining the HashSet of Certificate's issuer name to keep track of duplicates 
-                HashSet<string> issuerNameHashSet = new HashSet<string>();
-
-                //Enumerate Certificates from LocalMachine and CurrentUser root store 
-                AddX509Names(nameStack, StoreLocation.LocalMachine, issuerNameHashSet);
-                AddX509Names(nameStack, StoreLocation.CurrentUser, issuerNameHashSet);
-
-                Ssl.SslCtxSetClientCAList(context, nameStack);
```

Comment:
You can also remove this function and all it's definition and declaration statements from the native pal layer, if there are no other uses for this function.

Reply:
done